userspace/falco/falco.cpp

```diff
@@ -158,6 +158,8 @@ static void usage()
 	   "                               This causes every single line emitted by falco to be flushed,\n"
 	   "                               which generates higher CPU usage but is useful when piping those outputs\n"
 	   "                               into another process or into a script.\n"
+	   " -u                            Flip the inspector code to parse from userspace. This can be used in conjunction with\n"
+	   "                               the ptrace(2) based capture source pdig.\n"
```

Comment:
```suggestion
	   "                               To be used in conjunction with the ptrace(2) based driver (pdig).\n"
```

Continuing from above.

I think it's important to use the "driver" word as per our guidelines to avoid any confusion in the users.